src/main/java/org/broadinstitute/hellbender/engine/filters/UMIReadFilter.java

```diff
@@ -0,0 +1,42 @@
+package org.broadinstitute.hellbender.engine.filters;
+
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.hellbender.cmdline.ReadFilterArgumentDefinitions;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
+public class UMIReadFilter extends ReadFilter {
+    static final long serialVersionUID = 1L;
+    public static final String UMI_TAG = "RX";
```

Comment:
Reference `SAMTag.RX.name()` here instead of totally redefining it from scratch.  It's ok to still assign it to a named constant since RX.name() isn't super helpful..